tools/bin/sonarqube.sh

```diff
@@ -4,7 +4,7 @@ DIR=$(dirname "$0")
 SONAR_HOST=https://sonarcloud.io
 ORG_KEY=cmsgov
 
-if [[ "$CIRCLE_BRANCH" == "master" || ( ! -z $SONAR_OTHER_BRANCH && "$CIRCLE_BRANCH" == "$SONAR_OTHER_BRANCH" ) ]]; then
+#if [[ "$CIRCLE_BRANCH" == "master" || ( ! -z $SONAR_OTHER_BRANCH && "$CIRCLE_BRANCH" == "$SONAR_OTHER_BRANCH" ) ]]; then
```

Comment:
This prevents the preview from running and getting feedback from SonarQube in our PR and instead does a full scan with the results being uploaded to our SonarQube project.  Is that what you meant to happen?

Reply:
I was just going to ask that.  I'm not sure that's a bad thing, necessarily.  However, we should maybe limit this to just PRs otherwise we'll get pretty wild results over on the sonarcloud website.

Reply:
The problem with doing a full scan on a PR is that it is harder to see the results on the full SonarQube server.  For example, you'll need to somehow filter to your code changes only.  Also, you'll need to look up your results before another PR build because I think we loose the previous results.  Does anyone know of a way to go back to see the results of a scan from a previous time?

Reply:
I was testing the publish, removed the comments